lib/tools/src/fprof.erl

```diff
@@ -817,13 +817,13 @@ Option description:
 - **`start`** - Clears all tracing from the node and starts a new `fprof` trace.
   Either option `start` or `stop` must be specified, but not both.
 
-- **`verbose` | `{verbose, boolean()}`** - The options `verbose` or
-  `{verbose, true}` adds some trace flags that `fprof` does not need, but that
+- **`verbose` | `{verbose, boolean()}`** - The option `verbose` (or
+  `{verbose, true}`) adds some trace flags that `fprof` does not need, but that
   can be interesting for general debugging purposes. This option is only allowed
   with the `start` option.
 
-- **`cpu_time` | `{cpu_time, boolean()}`** - The options `cpu_time` or
-  `{cpu_time, true}` makes the timestamps in the trace be in CPU time instead of
+- **`cpu_time` | `{cpu_time, boolean()}`** - The option `cpu_time` (or
+  `{cpu_time, true}`) makes the timestamps in the trace be in CPU time instead of
   wallclock time which is the default. This option is only allowed with the
   `start` option.
```

Comment:
```suggestion
- **`cpu_time` | `{cpu_time, boolean()}`** - The `cpu_time` or
  `{cpu_time, true}` options make the timestamps in the trace be in CPU time instead of
  the default wallclock time. These options are only allowed with the
  `start` option.
```